frontend/app/configuration/Social.scala

```diff
@@ -0,0 +1,8 @@
+package configuration
+import configuration.Config
+
+object Social {
+  val youtube = "https://www.youtube.com/user/guardianmembership"
+  val googleplus = "https://plus.google.com/+guardianmembership/"
+  val twitter = s"http://www.twitter.com/${Config.twitterUsername}"
```

Comment:
I think we could probably remove `twitterUsername` as config too, but that can be left to another PR if you like!